pkg/service/elk_service.go

```diff
@@ -150,12 +158,51 @@ func (k *Kibana) SyncConfig(globalCtx *GlobalCtx) *executor.TransferCtx {
 	position := []executor.Position{
 		{LocalDir: genCfg, RemoteDir: filepath.Join(k.spec.RemoteCfgPath, "kibana.yml")},
 	}
+	if k.spec.ProvisioningTemplate != "" {
+		position = append(position, executor.Position{LocalDir: fmt.Sprintf("template/kibana/%s.ndjson", k.spec.ProvisioningTemplate),
```

Comment:
spec.ProvisionTemplate should already contain the `.ndjson` suffix

Reply:
thx ef514cb